src/ansys/dpf/core/helpers/utils.py

```diff
@@ -48,3 +48,19 @@ def _sort_supported_kwargs(bound_method, **kwargs):
         warnings.warn(txt)
     # Return the accepted arguments
     return kwargs_in
+
+
+def indent(text, subsequent_indent="", initial_indent=None):
```

Comment:
@Matteo-Baussart-ANSYS this will need a docstring with description and parameters. Also typehinting?